src/main/java/frc/robot/RobotContainer.java

```diff
@@ -154,12 +154,10 @@ private void configureBindings() {
                 () -> {
                   drivetrain.configNeutralMode(NeutralModeValue.Coast);
                   elevator.setNeutralMode(NeutralModeValue.Coast);
+                  drivecommand.stopDrivetrain();
```

Comment:
Two problems
1: name is wrong  - should be driveCommand
2: this does not actually execute the command.  It is returning a Command object that gets ignored.
Try instead adding`.andThen(driveCommand.stopDrivetrain())`